docs/tempo/website/configuration/_index.md

```diff
@@ -456,11 +458,28 @@ compactor:
 ```
 
 ## Storage
+
+Tempo supports Amazon S3, GCS, Azure, local file system, and Parquet for storage. In addition, you can use Memcached or Redis for increased query performance.
+
 For more information on configuration options, see [here](https://github.com/grafana/tempo/blob/main/tempodb/config.go).
 
-The storage block is used to configure TempoDB. It supports S3, GCS, Azure, local file system, and optionally can use Memcached or Redis for increased query performance.
+### Local storage recommendations
+
+While you can use local storage, object storage is recommended for production workloads.
+Tempo is designed for object storage more than local storage.
```

Comment:
The main reason for this is that a local backend will not correctly retrieve traces with a distributed deployment unless all components have access to the same disk. I should have mentioned this in the slack thread, we should add it here.

Reply:
Added to the content.  Thank you.